src/game/GameScenario.ts

```diff
@@ -324,12 +324,47 @@ export class BasicGameScenario implements GameScenario {
         return array[selectionIndex]
     }
 
+    private _cardDataToCard<T extends CardData | EventCard>(
+        data: T,
+    ): Card<WorldState> {
+        return {
+            image: data.image ?? '',
+            title: data.title ?? '',
+            text: data.text ?? '',
+            location: data.location ?? '',
+            match: () => true,
+            weight: data.weight ?? 1,
+            actions: {
+                left: {
+                    description: data.actions.left.description ?? 'No',
+                    modifier: (state) => {
+                        return this.getUpdatedState(
+                            state,
+                            data,
+                            data.actions.left,
+                        )
+                    },
+                },
+                right: {
+                    description: data.actions.right.description ?? 'No',
```

Comment:
This default value should be "Yes"

Reply:
Also interesting how this data seems to override `CardView.tsx`. Maybe this is supposed to be changed in the next PR #157 ?